src/ansi.rs

```diff
@@ -1551,6 +1554,28 @@ where
 
                 handler.clear_line(mode);
             },
+            ('k', [b' ']) => {
+                // SCP control
+                //
+                // Changes from ECMA-48 as proposed by the BiDi draft proposal:
+                //  * Recognize value 0 for Ps1
+                //  * Allow passing no params or Ps1 alone
+                //  * Assume default 0 value for both Ps1 and Ps2
+                //
+                //  https://terminal-wg.pages.freedesktop.org/bidi/recommendation/escape-sequences.html
+                let ps1 = params_iter.next().unwrap_or(&[0]);
+                let ps2 = params_iter.next().unwrap_or(&[0]);
+                if params_iter.next().is_some() {
+                    unhandled!();
+                } else {
+                    match (ps1, ps2) {
+                        (&[ps1], &[ps2]) if ps1 <= 2 && ps2 <= 2 => {
```

Comment:
Arguably this is a restriction that doesn't benefit the consumer of VTE, since they can always deny values out of range themselves but once we deny them, they can never handle them.

Quickly looking at other escapes, it seems we mostly only error on unknown values when we're mapping to enums, which kinda makes sense to me. So I'd do the same here: Either map in an enum, or just pass values. Enums could be nice for documentation purposes.